SumBenchDockerfile

```diff
@@ -0,0 +1,14 @@
+FROM clipper/py-rpc:latest
+
+COPY containers/python/sum_container.py /containers/python/
+COPY bench/setup_sum_bench_docker.sh /bench/
+COPY bench/set_bench_env_vars.sh /bench/
+COPY containers/python/rpc.py /containers/python/
```

Comment:
Ahh I see the problem. RPCDockerfile copies the `rpc.py` to `/containers`, not `/containers/python` (https://github.com/ucbrise/clipper/blob/develop/RPCDockerfile#L13)

If you fix the import in `sum_container.py` you shouldn't need to copy `rpc.py`